docs/integrations/python.md

```diff
@@ -60,6 +60,7 @@ production setting.
 Optionally, to enable communication via proxies, simply set the proxy configuration:
 
 ```python
+configuration.ssl_ca_cert = <ssl CA certificate path>
```

Comment:
path to a file of concatenated CA certificates in PEM format. Set this to customize the certificate file to verify the peer.